com.unity.render-pipelines.universal/Runtime/UniversalAdditionalCameraData.cs

```diff
@@ -337,18 +345,58 @@ public void SetRenderer(int index)
             m_RendererIndex = index;
         }
 
+        /// <summary>
+        /// Returns the selected scene-layers affecting this camera.
+        /// </summary>
```

Comment:
Extra documentation that bothered me when looking at the code.

Reply:
It's a bit vague IMO. In HDRP this field is documented like this:
> Layer mask used to select which volumes will influence this camera.

Reply:
@oleks-k I like the suggestion here. What are your thoughts?

Reply:
I would suggest this description (does it return a list?):

Returns the list of Layers selected in the Layer property of this Camera.

Is it technically correct?